sdk/standbypool/arm-standbypool/review/arm-standbypool.api.md

```diff
@@ -78,22 +76,21 @@ export enum KnownCreatedByType {
 
 // @public
 export enum KnownOrigin {
-    System = "system",
-    User = "user",
-    UserSystem = "user,system"
+    "user,system" = "user,system",
```

Comment:
share tsp link with me.

Reply:
this is from arm definition and it's readonly in response
```
/**
 * The intended executor of the operation; as in Resource Based Access Control (RBAC) and audit logs UX. Default value is "user,system"
 */
union Origin {
  /** Indicates the operation is initiated by a user. */
  user: "user",

  /** Indicates the operation is initiated by a system. */
  system: "system",

  /** Indicates the operation is initiated by a user or system. */
  `user,system`: "user,system",

  string,
}
```

Reply:
`user,system` is not easy to show the intellisense.